docs/maintaining-translations.md

```diff
@@ -0,0 +1,185 @@
+- `ignore-optional-plural`: The translation system of Matomo doesn't support plurals
+
+##### Enforced checks (`enforced_checks`)
+```json
+["php_format"]
+```
+
+##### Push on commit (`push_on_commit`)
+
+This is enabled, which means that whenever a change in Weblate is commited, a new pull request will be created or the existing one will be updated.
+
+##### Language code Style (`language_code_style`)
+
+We use "BCP style using hyphen as a separator" (`bcp`) as it is the closest to the language code style used in Matomo
+
+##### Adding new translation (`new_lang`)
+
+As a few steps need to be done to add a new language (see above), users can only request new languages, but don't add them. (`contact`)
+
+#### Add-ons
+
+In addition, the following add-ons are added to components:
+
+##### Add missing languages ([`weblate.consistency.languages`](https://docs.weblate.org/en/latest/admin/addons.html#addon-weblate-consistency-languages))
+
+This component is project-wide and adds languages added to one component to all others.
+
+##### Remove blank strings ([`weblate.cleanup.blank`](https://docs.weblate.org/en/latest/admin/addons.html#addon-weblate-cleanup-blank))
+
+##### Cleanup translation files ([`weblate.cleanup.generic`](https://docs.weblate.org/en/latest/admin/addons.html#addon-weblate-cleanup-generic))
+
+Removes translation keys no longer in `en.json`
+
+##### Customize JSON output ([`weblate.json.customize`](https://docs.weblate.org/en/latest/admin/addons.html#addon-weblate-json-customize))
+
+```json
+{
+    "sort_keys": true,
+    "indent": 4,
+    "style": "spaces"
+}
+```
+
+##### Squash Git commits ([`weblate.git.squash`](https://docs.weblate.org/en/latest/admin/addons.html#addon-weblate-git-squash))
+
+```json
+{
+    "squash": "language",
+    "append_trailers": true,
+    "commit_message": ""
+}
+```
+
+### List of involved translators
+
+On https://matomo.org/translations we are showing which languages Matomo is available in, as well as a list of translators for each language. This list is actually created through a Matomo API request (`LanguagesManager.getAvailableLanguagesInfo`). Therefore, the list of translators is stored within the base translations files (`General_TranslatorName`).
+
+Note: As the list is maintained in the translation files, the list on matomo.org will only be updated, once the translations have been update on GitHub, a new version was released and deployed.
+
+
+## Update data from Unicode CLDR
+
+The translations of the *Intl* plugin are automatically converted from the Unicode Common Locale Data Repository. To update them, check https://github.com/unicode-org/cldr-json/releases for the latest stable release of CLDR and update `$CLDRVersion` in ` plugins/Intl/Commands/GenerateIntl.php`. Afterwards you can run `php console translations:generate-intl-data` and commit the changed files in `plugins/Intl/lang`.
```

Comment:
@sgiehl this one we can remove as we have an internal process document for it

Reply:
Won't hurt to have it here. Also this is required to run when adding a new language, as the Intl data is required in this case. See https://github.com/matomo-org/developer-documentation/pull/523/files#diff-3e2d80ac2142428f04b2ca2e431263b92db4ea347f9e2f56e3451fc50d810d4fR96